### PR TITLE
[fix/PLAYER-5241] Issue with buffered part and played part in full sc…

### PR DESCRIPTION
**Pull request by ALexanderSPB**
Issue with buffered part and played part in full screen

This issue has appeared after mixins/ResizeMixin removal, so I added its functionality to the scrubberBar component.

https://jira.corp.ooyala.com/browse/PLAYER-5241

### CODE REVIEW

File: js/components/scrubberBar.jsx [hunk 1]
```diff
@@ -49,9 +49,13 @@ class ScrubberBar extends React.Component {
   componentWillReceiveProps(nextProps) {
     const { transitionedDuringSeek } = this.state;
     const { seeking } = nextProps;
+    const { componentWidth } = this.props;
     if (transitionedDuringSeek && !seeking) {
       this.setState({ transitionedDuringSeek: false });
     }
+    if (nextProps.componentWidth !== componentWidth) {
+      this.handleResize(nextProps);
```
Comment (KseniiaSokolovaEpam) [inline js/components/scrubberBar.jsx]: Do we need to add a test?

Comment (ALexanderSPB) [reply js/components/scrubberBar.jsx]: Added

Comment (ooyala-amohnatkin) [inline js/components/scrubberBar.jsx]: `handleResize` doesn't take arguments

File: tests/components/scrubberBar-test.js [hunk 2]
```diff
@@ -557,4 +557,23 @@ describe('ScrubberBar', function() {
...
+    const wrapper = Enzyme.mount(
+      <ScrubberBar
+        controller={mockController}
+        componentWidth={600}
+      />
+    );
+    const handleResize = jest.spyOn(wrapper.instance(), 'handleResize');
+    wrapper.setProps({ componentWidth: 1900 });
+
+    expect(handleResize).toHaveBeenCalled();
```
Comment (ooyala-amohnatkin) [inline tests/components/scrubberBar-test.js]: please use blackbox testing and check actual result, not that some inner method has been called

Comment (ooyala-amohnatkin) [reply tests/components/scrubberBar-test.js]: I suppose you should check `playheadPaddingStyle`

**Pull request merged**